sdk/keyvault/keyvault-secrets/README.md

```diff
@@ -85,16 +86,35 @@ Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to creat
   az keyvault show --name <your-key-vault-name>
   ```
 
+## Key concepts
+
+- The **SecretsClient** is the primary interface to interact with Azure Key Vault
+  from a JavaScript application. Once initialized, it provides a basic set of methods
+  that can be used to create, read, update and delete secrets.
+- Secret **versions** are one or more instances of the same unique secret name.
+  Each time a user assigns a value to a unique secret name, a new **version**
+  of that secret is created. Retrieving a secret by a name will always return
+  the latest value assigned, unless a specific version is provided to the
+  query.
+- If the Key Vault has
+  [soft-delete](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-ovw-soft-delete)
+  enabled, deleted secrets won't be removed, they will be marked as "deleted"
+  instead, so they can be retrieved or recovered anytime. A deleted secret can
+  only be accessed through specific functions for deleted secrets. Deleted
+  secrets can't be updated, they can only be either read, recovered or purged.
+- A **backup** can be generated from any created secret. These backups come as
```

Comment:
A **Key Vault backup** can be generated...

Reply:
It's a secret backup though, but I understand that this is about the format.